packages/headless/src/api/knowledge/post-answer-evaluation.ts

```diff
@@ -0,0 +1,31 @@
+import {answerSlice} from './answer-slice';
+
+export interface PostEvaluationArgs {
```

Comment:
I think I will change this to `EvaluationPOSTParams`

Reply:
Really not a big deal, and really more of an FYI, but we usually split API params into interfaces and compose request param types from those.

E.g.:

- https://github.com/coveo/ui-kit/blob/master/packages/headless/src/api/commerce/commerce-api-params.ts (defining and exporting the small param interfaces)
- https://github.com/coveo/ui-kit/blob/master/packages/headless/src/api/commerce/common/request.ts (composing request param types with those interfaces)

The advantage is we can reuse the same param definitions to compose multiple request param types in a given API.

That being said, I don't think there'd be much gain (if any) in doing that in this specific case, but if you find yourself duplicating param definitions, you can use this pattern!

Reply:
That is good to know!  yes, if at some point we have to reuse those param, I think we better follow this pattern indeed.